ion-schema/src/schema.rs

```diff
@@ -324,13 +323,12 @@ mod schema_tests {
         let mut resolver = Resolver::new(vec![]);
 
         // create a isl from owned_elements and verifies if the result is `ok`
-        let isl =
-            resolver.isl_schema_from_elements(IslVersion::V1_0, owned_elements, "my_schema.isl");
-        assert!(isl.is_ok());
+        let isl_result = resolver.isl_schema_from_elements(owned_elements, "my_schema.isl");
+        assert!(isl_result.is_ok());
 
+        let isl = isl_result.unwrap();
```

Comment:
minor suggestion—it seems strange to have an `assert!` followed by an `unwrap`. Why not just unwrap right away? (Or even better, use `expect(...)` with a message to make the test code more self-describing and the failure message more helpful for future developers.)
```suggestion
        let isl = isl_result.expect("<Thing we're checking> should be <condition we're checking for>");
```